ros2bag/test/test_play_qos_profiles.py

```diff
@@ -82,3 +82,4 @@ def test_qos_incomplete(self):
         expected_string_regex = re.compile(ERROR_STRING)
         matches = expected_string_regex.search(bag_command.output)
         assert not matches, print('ros2bag CLI did not produce the expected output')
+        
```

Comment:
should revert this change - may want to set your editor to always ensure newlines if you can